recipes/paraprobe/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "paraprobe" %}
+{% set version = "v0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://gitlab.com/{{ name }}/{{ name }}-toolbox/-/archive/{{ version }}/{{ name }}-toolbox-{{ version }}.tar.gz
+    sha256: b7aba8d5d56117390ba6231c4599131019090a919ec48f84f556f49fdc41057a
+    folder: paraprobe-toolbox
+    patches:
+      - paraprobe.patch
+  - fn: v0.4.6.tar.gz
+    url: https://github.com/chr1shr/voro/archive/refs/tags/v0.4.6.tar.gz
+    folder: voro++
+    sha256: e1eb9c1f5a393604f2ca56e935b7e850644f7a3a18f5a85a5b2617f51fc70343
+
+build:
+  number: 0
+  skip: True  # [win]
+  #skip: True  # [osx]
+  #skip: True  # [py<38] 
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+    - git
+    - m4
+
+  host:
+    - python
+    - cython
+    - setuptools
+    - {{ mpi }}
+    - gmp
+    - mpfr
+    - libhwloc
+    - libgomp
+    - hdf5
+    - hdf5 * {{ mpi_prefix }}_*
+    - tetgen
+    - fftw
+    - boost
+    - eigen
+    - voro
+    - cgal
+    - libaec
+    - numpy >=1.17
+  run:
+    - python
+    - setuptools
+    - scikit-learn ==1.0.2
+    - ase ==3.22.1
+    - ipywidgets ==7.6.3
+    - h5py ==3.6.0
+    - jupyterlab[full] ==3.0.0
+    - punx ==0.3.0
+    - nexpy ==0.14.4
+    - silx[full] ==1.0.0
+    - numpy >=1.17
+
+test:
+  imports:
+    - paraprobe_parmsetup
+    - paraprobe_transcoder
```

Comment:
Can you add tests for the executable? At least call `--help` on the executables or something simple.

Reply:
I tried this, but its a bit tricky. The executable has no help command, and it generates an error when called without an input file. The input data I have at the moment is too big, 100+ MBs. I have raised some issues: https://gitlab.com/paraprobe/paraprobe-toolbox/-/issues/30 https://gitlab.com/paraprobe/paraprobe-toolbox/-/issues/31